src/sentry/api/endpoints/project_group_index.py

```diff
@@ -455,6 +454,10 @@ def put(self, request, project):
                             group=group,
                             reason=GroupSubscriptionReason.status_change,
                         )
+                        if not GroupAssignee.objects.filter(
+                            group=group, project=group.project
```

Comment:
I'm going to guess that `group.project` has already been loaded from the database, but good practice here is to use `project_id=group.project_id` unless you absolutely need the `Project` instance for some reason.

There's also no index that covers both of these fields, so you can just restructure this for simplicity to `if not group.assignee_set.exists()`.